tests/php/ORM/SQLSelectTest.php

```diff
@@ -24,18 +23,6 @@ class SQLSelectTest extends SapphireTest
 
     protected $oldDeprecation = null;
 
-    protected function setUp(): void
```

Comment:
Removed because they called functionality that was previously removed from Deprecation